e2e_tests/.env.example

```diff
@@ -23,12 +23,17 @@ QABOX_DUO_EMAIL=
 QABOX_DUO_CODE=
 
 # The URL APP ID that is used to connect to the backend
-QABOX_SERVER=www.binaryqa10.com
-QABOX_APP_ID=1013
+QABOX_SERVER=qa10.deriv.dev
+QABOX_APP_ID=9999
```

Comment:
`9999` is the appid for `https://localhost.binary.sx/`.
Are we changing it in QA10?

Reply:
What should it be?
I checked it with Mitra from qa team and I didn't get the right answer for that!